examples/window-app/silabs/efr32/include/WindowManager.h

```diff
@@ -0,0 +1,251 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app/clusters/window-covering-server/window-covering-server.h>
+#include <lib/core/CHIPError.h>
+
+#include <FreeRTOS.h>
+#include "LEDWidget.h"
+
+#include <queue.h>
+#include <setup_payload/QRCodeSetupPayloadGenerator.h>
+#include <string>
+#include <task.h>
+#include <timers.h>
+#ifdef DISPLAY_ENABLED
+#include <LcdPainter.h>
+#endif
+
+using namespace chip::app::Clusters::WindowCovering;
+
+class WindowManager
+{
+public:
+
+    static WindowManager sInstance;
+    struct Timer
```

Comment:
Is this whole struct needed?

Reply:
It's the Timer struct from WindowApp (base class) and WindowAppImpl combined, since WindowAppImpl extended the definition of the base class Timer struct.